pkg/reconcile/pipeline/context/application.go

```diff
@@ -1,10 +1,11 @@
 package context
 
 import (
+	"reflect"
+
 	"github.com/redhat-developer/service-binding-operator/api/v1alpha1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"reflect"
```

Comment:
no-op change: only import order changed - please revert.

Reply:
I have integrated [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) with my editor. So, this change happens when I save the file. Can I send a separate PR with these changes?

Reply:
but this is the only change on this file, not sure why you needed to save it? Please revert it. If we want to fix import ordering globally, that should be done in a separate PR, but we should agree about the common rules that will be used by majority of IDEs/editors we use.

Reply:
I send separate PR: https://github.com/redhat-developer/service-binding-operator/pull/981

Reply:
Reverted here: 6ee0d5dedefe8d356fcee88eebd9fea701325b79